scripts/deploy/package-memorypipeline.sh

```diff
@@ -86,7 +86,16 @@ if [[ ! -d "$PUBLISH_ZIP_DIRECTORY" ]]; then
 fi
 
 echo "Build configuration: $CONFIGURATION"
-dotnet publish "$SCRIPT_ROOT/../../memorypipeline/CopilotChatMemoryPipeline.csproj" --configuration $CONFIGURATION --framework $DOTNET --runtime $RUNTIME --self-contained --output "$PUBLISH_OUTPUT_DIRECTORY" /p:AssemblyVersion=$VERSION /p:FileVersion=$VERSION /p:InformationalVersion=$INFO
+dotnet publish "$SCRIPT_ROOT/../../memorypipeline/CopilotChatMemoryPipeline.csproj" \
+    --configuration $CONFIGURATION \
+    --framework $DOTNET \
+    --runtime $RUNTIME \
+    --self-contained \
+    --output "$PUBLISH_OUTPUT_DIRECTORY" \
+    //p:AssemblyVersion=$VERSION \
+    //p:FileVersion=$VERSION \
+    //p:InformationalVersion=$INFO
```

Comment:
This should not be commented out.

If this is what is giving us problems, maybe we just need to surround the arguments with quotes? (Ex: "$VERSION"

Reply:
These are not comments. Bash scripts use `#` for comments. An additional `/` is needed here to escape the following `/`.

Reply:
Ah!  I see, it's so Bash doesn't interpret what follows the / as a path...

I'm still a bit dazed by this ,but OK! :)